README.md

```diff
@@ -94,6 +95,68 @@ Default: `{}`
 
 A cache of key/value pairs to used to seed the manifest. This may include a set of [custom key/value](https://developer.mozilla.org/en-US/Add-ons/WebExtensions/manifest.json) pairs to include in your manifest, or may be used to combine manifests across compilations in [multi-compiler mode](https://github.com/webpack/webpack/tree/master/examples/multi-compiler). To combine manifests, pass a shared seed object to each compiler's ManifestPlugin instance.
 
+### `filter`
+
+Type: `function`
+
+Filter out files.
+
+
+### `map`
+
+Type: `function`
+
+Modify files details before the manifest is created.
+
+### `reduce`
+
+Type: `function`
+
+Create the manifest. It must return an `Object`.  The default one will take the name of the chunk as a key and the path of the file as the value.
```

Comment:
As I understand, we should note here: «custom reduce opt function replaces that default `reduce` logic:
```js
function (manifest, file) {
        manifest[file.name] = file.path;
        return manifest;
}
```